pulp_container/app/viewsets.py

```diff
@@ -642,33 +696,29 @@ class ContainerDistributionViewSet(BaseDistributionViewSet):
                 "action": ["create"],
                 "principal": "authenticated",
                 "effect": "allow",
-                "condition": [
-                    "has_model_perms:container.add_containerdistribution",
-                    "has_manage_namespace_dist_perms:container.manage_namespace_distributions",
-                ],
```

Comment:
Having no condition here, means every authenticated user can create whatever not yet existing distribution (even in existing namespaces).
It should be
(namespace does not exist) or (user.has_namespace_perm(create_distribution))

Reply:
I added the condition back